openid-sse-framework-1_0.xml

```diff
@@ -887,7 +775,342 @@ Cache-Control: no-store
+            MUST be updated by the Transmitter. Any properties missing in the request
+            MUST NOT be changed by the Transmitter.</t>
+
+            <t>Read-only properties beside the stream_id MAY be present,
+            but they MUST match the expected value. Missing read-only
+            properties will be ignored by the Transmitter.</t>
+
+            <t>The following is a non-normative example request to replace an Event Stream’s
+              configuration:</t>
+
+            <figure title="Example: Update Stream Configuration Request"
+              anchor="figupdateconfigreq">
+              <artwork><![CDATA[
+PATCH /sse/stream HTTP/1.1
+Host: transmitter.example.com
+Authorization: Bearer eyJ0b2tlbiI6ImV4YW1wbGUifQo=
+
+{
+  "stream_id": "f67e39a0a4d34d56b3aa1bc4cff0069f",
+  "iss": "https://tr.example.com",
+  "events_requested": [
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3",
+    "urn:example:secevent:events:type_4"
+  ],
+}
+]]></artwork>
+            </figure>
+
+            <t>The following is a non-normative example response:</t>
+
+            <figure title="Example: Update Stream Configuration Response"
+              anchor="figupdateconfigresp">
+              <artwork><![CDATA[
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+{
+  "stream_id": "f67e39a0a4d34d56b3aa1bc4cff0069f",
+  "iss": "https://tr.example.com",
+  "aud": [
+    "http://receiver.example.com/web",
+    "http://receiver.example.com/mobile"
+  ],
+  "delivery": {
+    "delivery_method":
+      "https://schemas.openid.net/secevent/risc/delivery-method/push",
+    "url": "https://receiver.example.com/events"
+  },
+  "events_supported": [
+    "urn:example:secevent:events:type_1",
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3"
+  ],
+  "events_requested": [
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3",
+    "urn:example:secevent:events:type_4"
+  ],
+  "events_delivered": [
+    "urn:example:secevent:events:type_2",
+    "urn:example:secevent:events:type_3"
+  ]
+}
+]]></artwork>
+            </figure>
+
+            <t>Pending conditions or errors are signaled with HTTP status codes as follows:</t>
+
+            <texttable title="Update Stream Configuration Errors" anchor="tabupdateconfig">
+              <ttcol align="left">Code</ttcol>
+              <ttcol align="left">Description</ttcol>
+              <c>202</c>
+              <c>if the update request has been accepted, but not processed. Receiver MAY try the
+                same request later in order to get processing result.</c>
+              <c>400</c>
+              <c>if the request body cannot be parsed, a read-only property is incorrect, or if the request is otherwise invalid</c>
+              <c>401</c>
+              <c>if authorization failed or it is missing</c>
```

Comment:
"if credentials are missing or invalid"

Reply:
I'd prefer to leave these consistent with the other error definitions in the spec. We can create a new issue to discuss changing all of them at once.